### PR TITLE
Make ManhwaZ a multisrc + Add UmeTruyen

### LINKED ISSUE #1370: UmeTruyen
### Source name

UmeTruyen

### Source link

https://umetruyenvip.com

### Source language

Vietnamese

### Other details

18+/NSFW = yes

### Acknowledgements

- [X] I have checked that the extension does not already exist by searching the [extension listing](https://keiyoushi.github.io/extensions/) and verified it does not appear there.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a meaningful title with the source name.
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by beer-psi**
Closes #1370 

Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio
- [x] Have removed `web_hi_res_512.png` when adding a new extension

### CODE REVIEW

File: lib-multisrc/manhwaz/src/eu/kanade/tachiyomi/multisrc/manhwaz/ManhwaZ.kt [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+    @OptIn(DelicateCoroutinesApi::class)
+    private fun fetchGenreList() {
+        if (fetchGenreStatus != FetchGenreStatus.NOT_FETCHED || fetchGenreAttempts >= 3) {
+            return
+        }
+
+        fetchGenreStatus = FetchGenreStatus.FETCHING
+        fetchGenreAttempts++
+
+        GlobalScope.launch(Dispatchers.IO) {
```
Comment (AwkwardPeak7) [inline lib-multisrc/manhwaz/src/eu/kanade/tachiyomi/multisrc/manhwaz/ManhwaZ.kt]: CoroutineScope?

File: src/en/manhwaz/build.gradle [hunk 2]
```diff
@@ -1,9 +1,9 @@
 ext {
     extName = 'ManhwaZ'
-    extClass = '.ManhwaZ'
-    themePkg = 'madara'
+    extClass = '.ManhwaZCom'
+    themePkg = 'manhwaz'
     baseUrl = 'https://manhwaz.com'
-    overrideVersionCode = 0
+    overrideVersionCode = 34
```
Comment (AwkwardPeak7) [inline src/en/manhwaz/build.gradle]:
```suggestion
    overrideVersionCode = 35
```

Comment (beerpiss) [reply src/en/manhwaz/build.gradle:6]: ManhwaZ is currently 1.4.34, so baseVersionCode (1) + overrideVersionCode (34) will make it 35.

File: lib-multisrc/manhwaz/src/eu/kanade/tachiyomi/multisrc/manhwaz/ManhwaZ.kt (hunk 1 above)
Comment (beer-psi) [reply lib-multisrc/manhwaz/src/eu/kanade/tachiyomi/multisrc/manhwaz/ManhwaZ.kt]: Didn't we talk about not using CoroutineScope before? https://github.com/keiyoushi/extensions-source/pull/1292#discussion_r1493317304

Comment (beer-psi) [reply lib-multisrc/manhwaz/src/eu/kanade/tachiyomi/multisrc/manhwaz/ManhwaZ.kt]: Ah nevermind, you fixed it to use a CoroutineScope already.

**Review by AwkwardPeak7 — APPROVED** [review]

**Pull request merged**